_quarto.yml

```diff
@@ -9,8 +9,8 @@ project:
   output-dir: docs
 
 book:
-  title: "write your project title here"
-  author: ["student 1 first and last name", "student 2 first and last name", "student 3 first and last name", "student 4 first and last name"]
+  title: "Exploring Baseball"
+  author: ["Huthaifa Mohammad, Sidney Langford, Suweda Said, Basir Talayee, Harlan Grady"]
```

Comment:
I think the name of each person need to be surrounded by double quotation marks in order to appear as separate lines.